From 900a2ab3540520ba97e79bb39d78017c8d296630 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynhnguyen.xb@renesas.com>
Date: Thu, 15 Oct 2020 17:03:21 +0700
Subject: [PATCH 449/475] ARM: dts: iwg22d-sodimm: Support USB OTG feature

Add support for USB OTG on the iWave R8A7745 (RZ/G1E) board.

Both USB Host (pci0) and Function (USBHS) drivers are enabled.
The USB PHY driver determines which IP block should be connected
based on vbus and id signals read via gpios.

Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 arch/arm/boot/dts/r8a7745-iwg22d-sodimm.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7745-iwg22d-sodimm.dts b/arch/arm/boot/dts/r8a7745-iwg22d-sodimm.dts
index 8a3bab3..2f7470b 100644
--- a/arch/arm/boot/dts/r8a7745-iwg22d-sodimm.dts
+++ b/arch/arm/boot/dts/r8a7745-iwg22d-sodimm.dts
@@ -400,6 +400,10 @@
 
 &usbphy {
 	status = "okay";
+	renesas,id-gpio = <&gpio4 6 GPIO_ACTIVE_LOW>;
+	renesas,vbus-gpio = <&gpio5 25 GPIO_ACTIVE_LOW>;
+	renesas,vbus-pwr-gpio = <&gpio5 24 GPIO_ACTIVE_LOW>;
+	renesas,wakeup;
 };
 
 &vin1 {
-- 
2.7.4

